redisinsight/ui/src/pages/workbench/components/wb-results/WBResults/WBResults.tsx

```diff
@@ -84,6 +86,16 @@ const WBResults = (props: Props) => {
           activeResultsMode={activeResultsMode}
           resultsMode={resultsMode}
           onQueryOpen={() => onQueryOpen(id)}
+          onQueryProfile={(profileType: ProfileQueryType) => {
```

Comment:
could you please move this to the separate function?

Reply:
We are using a prop function of the component here. Are you suggesting we can define the function inside the component?

Reply:
yes, just a function inside the component
smth like that:

```
const handleQueryProfile = (profileType: ProfileQueryType) => {}
...
onQueryProfile={handleQueryProfile}
```